src/compile/scale/range.ts

```diff
@@ -233,6 +243,21 @@ export function defaultRange(
   throw new Error(`Scale range undefined for channel ${channel}`);
 }
 
+/**
+ * Returns the linear interpolation of the range according to the cardinality
+ *
+ * @param rangeMin start of the range
+ * @param rangeMax end of the range
+ * @param cardinality number of values in the output range
+ */
+function interpolateRange(rangeMin: number, rangeMax: number, cardinality: number) {
+  const ranges: number[] = [];
+  for (let i = 1; i <= cardinality; i++) {
+    ranges.push(rangeMin + (i * (rangeMax - rangeMin)) / cardinality);
```

Comment:
1) This formula won't include rangeMin

2) You can calculate step once instead of every time.

3) Step should be `(rangeMax - rangeMin)) / (cardinality - 1)`, so you can start from i = 0, 1, 2, ..., cardinality - 1

Reply:
I tried including `rangeMin`, but it turns out to be pretty small for default range of 1 in the case of `circle`

Reply:
minSize is `9` by default.  

> minSize: 9, // Point size is area. For square point, 9 = 3 pixel ^ 2, not too small!

Looks like your discretizing scales include `zero=true` by default.  I think you should change that (and update the docs).

Reply:
In any case, given the name interpolateRange, it's weird that the method skip the first point.  (Then it's not a typical interpolation function anymore.)